generators/aws-containers/prompts.js

```diff
@@ -1,39 +1,65 @@
 const _ = require('lodash');
 const chalk = require('chalk');
+const databaseTypes = require('jhipster-core').JHipsterDatabaseTypes.Types;
 
 const AURORA_DB_PASSORD_REGEX = /^[^@"\/]{8,42}$/; // eslint-disable-line
+const CLOUDFORMATION_STACK_NAME =  /[a-zA-Z][-a-zA-Z0-9]*/; // eslint-disable-line
+
+const SCALING_TO_CONFIG = {
```

Comment:
I think you should merge back the two objects and then move the `map` done in both scaling and performance prompts here as constants.
**Example:** `performanceLevels` from `promptPerformance` becomes `PERFORMANCE_PROMPT_CHOICES`.

Reply:
Just realised that we did those mapping in the function when there is no need to do so as we don't need anything from the running context